app/Models/DuesPackage.php

```diff
@@ -235,8 +235,8 @@ public function scopeActive(Builder $query): Builder
      */
     public function scopeAccessActive(Builder $query): Builder
     {
-        return $query->where('access_start', '<', date('Y-m-d H:i:s'))
-            ->where('access_end', '>', date('Y-m-d H:i:s'));
+        return $query->where('access_start', '<', now())
+            ->where('access_end', '>', now());
```

Comment:
Why these changes?

Reply:
It's just cleaner.

In other places where the timestamp portion was previously omitted, there is an actual functional change: `now()` is more precise.